tensorflow/contrib/learn/python/learn/learn_io/data_feeder.py

```diff
@@ -363,6 +363,11 @@ def __init__(self,
 
     num_samples = list(self._x.values())[0].shape[
```

Comment:
what do you exactly have in mind? I thought there are only 2 cases (NumPy array or Tensor).